vagrant-provision.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# prepare Debian system
+apt-get update && apt-get dist-upgrade -y
+apt-get install python-pip python-dev build-essential default-jre libxslt-dev libxml2-dev libffi-dev libssl-dev git curl -y
+
+# get nodejs from external sources (Debian is outdated)
+curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
```

Comment:
I know some people like `curl | sudo bash -` installers but from a security point of view they are a nightmare. Since  the Vagrantfile specifies a very specific debian version, it shouldn't be too hard to add the APT repo/key manually instead of running random code retrieved from the internet with `sudo`...